classes/module/Module.php

```diff
@@ -1982,28 +1982,45 @@ public function l($string, $specific = false, $locale = null)
     public function updatePosition($id_hook, $way, $position = null)
     {
         foreach (Shop::getContextListShopID() as $shop_id) {
-            $sql = 'SELECT hm.`id_module`, hm.`position`, hm.`id_hook`
+            $sqlQueryToGetAvailableHookPositions = 'SELECT hm.`id_module`, hm.`position`, hm.`id_hook`
```

Comment:
Your SQL variables could be less long if you inverted the `sql` part:

`$sqlQueryToGetXYZ` ➡️ `$getXYZSql`